pyrometheus/__init__.py

```diff
@@ -252,7 +252,7 @@ def troe_falloff_expr(react: ct.Reaction, t):
     troe_params = react.falloff.parameters
     troe_1 = (1.0-troe_params[0])*p.Variable("exp")(-t/troe_params[1])
     troe_2 = troe_params[0]*p.Variable("exp")(-t/troe_params[2])
-    if troe_params[3] > 1.0e-16:
+    if np.abs(troe_params[3]) > 1.0e-16:
```

Comment:
```suggestion
    if np.abs(troe_params[3]) > np.finfo(troe_params_dtype).eps:
```
I don't know if we expect `troe_params` to be anything other than double precision values, but I nonetheless find this form preferable because it makes the intention clear.

Reply:
> > I can add test on a 32-species Ethylene mechanism to back this up if needed.
> 
> IMO this would be a very good thing.
> 
> A point of view about software that I've adopted for myself is that anything that's not tested should be considered broken already. (Or rephrased more positively: any property you think you value in software should have an automated test to back it up.) So, if you'd like Troe fall-off reactions to continue to work... :)

I will put in the test.